specification/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2022-09-04/redhatopenshift.json

```diff
@@ -0,0 +1,1127 @@
+        },
+        "display": {
+          "$ref": "#/definitions/Display",
+          "description": "The object that describes the operation."
+        },
+        "origin": {
+          "description": "Sources of requests to this operation.  Comma separated list with valid values user or system, e.g. \"user,system\".",
+          "type": "string"
+        }
+      }
+    },
+    "OperationList": {
+      "description": "OperationList represents an RP operation list.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List of operations supported by the resource provider.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      }
+    },
+    "ProvisioningState": {
+      "description": "ProvisioningState represents a provisioning state.",
+      "enum": [
+        "AdminUpdating",
+        "Creating",
+        "Deleting",
+        "Failed",
+        "Succeeded",
+        "Updating"
+      ],
+      "type": "string"
+    },
+    "ServicePrincipalProfile": {
+      "description": "ServicePrincipalProfile represents a service principal profile.",
+      "type": "object",
+      "properties": {
+        "clientId": {
+          "description": "The client ID used for the cluster.",
+          "type": "string"
+        },
+        "clientSecret": {
+          "description": "The client secret used for the cluster.",
+          "type": "string"
+        }
+      }
+    },
+    "SoftwareDefinedNetwork": {
+      "description": "SoftwareDefinedNetwork is a choice of network used for Pods.",
+      "enum": [
+        "OVNKubernetes",
+        "OpenShiftSDN"
+      ],
+      "type": "string",
+      "x-ms-enum": {
+        "name": "SoftwareDefinedNetwork",
+        "modelAsString": true
+      }
+    },
+    "SyncSet": {
+      "description": "SyncSet represents a SyncSet",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/SyncSetProperties"
+        }
+      }
+    },
+    "SyncSetList": {
+      "description": "SyncSetList represents a list of SyncSets",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SyncSet"
+          },
+          "x-ms-identifiers": []
+        },
+        "nextLink": {
+          "description": "The link used to get the next page of operations.",
+          "type": "string"
+        }
+      }
+    },
+    "SyncSetProperties": {
+      "description": "SyncSet Properties represents the properties of a syncSet",
+      "type": "object",
+      "properties": {
+        "resources": {
+          "type": "object"
+        }
```

Comment:
free form objects are not allowed. Please define the schema for this

Reply:
I dont see that this comment was addressed

Reply:
@rkmanda @raosuhas can you expand on what you want here? What I'm seeing is that SyncSetProperties has a single "resources" string property. In the implementation, "resources" will be a base64-encoded blob.